packages/synapse-react-client/src/components/EntityFinder/EntityFinder.test.tsx

```diff
@@ -355,61 +361,24 @@ describe('EntityFinder tests', () => {
   })
 
   describe('Search', () => {
-    it('Updates the search button text when only one type is selectable', async () => {
```

Comment:
The search input field is no longer hidden behind a button press, so many of these search tests are no longer relevant.